src/ocean/time/timeout/model/ExpiryRegistrationBase.d

```diff
@@ -96,6 +96,20 @@ abstract class ExpiryRegistrationBase : IExpiryRegistration
 
         /***********************************************************************
 
+            Same as `unregister` but also removes expirty from list of currently
+            expired registrations. That means it can be called from `timeout`
+            callbacks, contrary to `unregister`.
```

Comment:
Could you extend the doc of `update` to explain why it should not be called from a `timeout` callback?

Also: should `unregister` be called at all by any non-library code any more? (If not, please document this as well.)

Reply:
> should unregister be called at all by any non-library code any more?

I was hoping you can answer that :) To me it seems 100% wrong to call it with old semantics but I don't have a good overview of how it is used across apps.